content/efcore/getting-started/samples/generating-databases-and-data-models/generate-database/Book.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
```

Comment:
This `using` also isn't needed. Sorry I didn't catch it in the last round of change requests.